packages/core/src/__tests__/utils.test.ts

```diff
@@ -101,44 +102,133 @@ describe('utils', () => {
       expect(getMatchingStatusCodeRange('2002')).toEqual('2002');
       expect(getMatchingStatusCodeRange(4000)).toEqual('4000');
     });
+  });
 
-    describe('isConfigFileExist', () => {
-      beforeEach(() => {
-        jest
-          .spyOn(fs, 'existsSync')
-          .mockImplementation((path) => path === 'redocly.yaml' || path === 'redocly.yml');
-        jest.spyOn(path, 'extname').mockImplementation((path) => {
-          if (path.endsWith('.yaml')) {
-            return '.yaml';
-          } else if (path.endsWith('.yml')) {
-            return '.yml';
-          } else {
-            return '';
-          }
-        });
-      });
+  describe('isValidURL', () => {
```

Comment:
I don't think we need so many tests for this. I would also not validate this at all. I would add a very basic URL validation.

Reply:
you're right, URL constructor is also more performant and reliable.